vhost-device-scmi/src/scmi.rs

```diff
@@ -575,6 +590,26 @@ pub trait ScmiDevice: Send {
         message_id: MessageId,
         parameters: &[MessageValue],
     ) -> Result<MessageValues, ScmiDeviceError>;
+
+    // Get device notify fd
+    fn get_notify_fd(&self) -> Option<RawFd>;
+
+    // Set id for this ScmiDeivce
+    fn set_id(&mut self, id: usize);
+
+    // Get id of this ScmiDevice
+    fn get_id(&self) -> usize;
+
+    /// Get notify message from this device.
+    ///
+    /// Usually need to refine it for different protocal.
+    /// device_index is the index of this device in its protocol list.
+    /// message_id points that which notification it should be sent.
```

Comment:
"message_id is the notification message id to be used." ?

Reply:
Oh, yes. For notification in sensor protocol, spec defines two commands.

> SENSOR_TRIP_POINT_EVENT -> message_id=0
> SENSOR_UPDATE -> message_id = 1

Currently, we only implement for "SENSOR_UPDATE".